test/routes/routesApi.spec.js

```diff
@@ -1155,4 +1155,48 @@ describe('Le serveur MSS des routes /api/*', () => {
         .catch((e) => done(e.response?.data || e));
     });
   });
+
+  describe('quand requête GET sur `/api/annuaire/suggestions`', () => {
+    beforeEach(() => {
+      testeur.referentiel().estCodeDepartement = () => true;
+    });
+
+    it('retourne une erreur HTTP 400 si le terme de recherche est vide', (done) => {
+      testeur.verifieRequeteGenereErreurHTTP(
+        400,
+        'Le terme de recherche ne peut pas être vide',
+        { method: 'get', url: 'http://localhost:1234/api/annuaire/suggestions?departement=75' },
+        done
+      );
+    });
+
+    it("retourne une erreur HTTP 400 si le département n'est pas dans le referentiel", (done) => {
+      testeur.referentiel().estCodeDepartement = () => false;
+      testeur.verifieRequeteGenereErreurHTTP(
+        400,
+        'Le département doit être valide (01 à 989)',
+        { method: 'get', url: 'http://localhost:1234/api/annuaire/suggestions?recherche=mairie&departement=990' },
+        done
+      );
+    });
+
+    it("utilise l'adaptateur annuaire pour renvoyer le résultat de la recherche", (done) => {
```

Comment:
Suggestion

`it("recherche les organisations correspondantes grâce à l'adaptateur annuaire", (done) => {`